lib/helpController.class.php

```diff
@@ -32,6 +32,7 @@ public function runStrategy()
   --savedir      directory to save schema.php and migrations
   --verbose      if value On script will output each executed query
   --versiontable table to keep db revisions new row for each revision
+  --versiontable-engine table engine for versiontable (MyISAM, InnoDB)
```

Comment:
Can you also indent other options? The new one does visually blend into --versiontable